vars/piperPipelineStageInit.groovy

```diff
@@ -70,7 +70,8 @@ import static com.sap.piper.Prerequisites.checkScript
      * Sets the build version.
      * @possibleValues `true`, `false`
      */
-    'artifactPrepareVersion'
+    'artifactPrepareVersion',
+    'transportRequestReqIDFromGit'
```

Comment:
Could you describe this parameter  in a comment above this?

Reply:
Please look at the new commit. Done as suggested